wire-ios-data-model/Source/Model/User/ZMUser.swift

```diff
@@ -52,6 +52,13 @@ extension ZMUser: UserType {
         return featureRepository.fetchDigitalSignature().status == .enabled
     }
 
+    var isMLSEnabled: Bool {
```

Comment:
```suggestion
    private func isMLSEnabled() -> Bool {
```
there's nothing from the ZMUser needed here, so as it might be tricky to move the accept the `accept(completion: @escaping (Error?) -> Void)` method out of the model, I would at least restrict the access to this method